drivers/filesystems/CMakeLists.txt

```diff
@@ -2,7 +2,9 @@
 add_subdirectory(btrfs)
 add_subdirectory(cdfs)
 add_subdirectory(ext2)
-#add_subdirectory(fastfat)
+if(SARCH STREQUAL "xbox")
+add_subdirectory(fastfat)
```

Comment:
I then wonder whether you could directly rename that fastfat directory into vfatfs, and the fastfat_new into fastfat. (Maybe in a separate commit?)